### ISSUE TITLE
adding capture in input field

### ISSUE THREAD

**Issue by pilere** [tomitrescak/meteor-uploads#166]
Hi,

I want to add capture in the input field :
input class="jqUploadclass" type="file" data-form-data="{"contentType":null}"
to
input class="jqUploadclass" type="file" data-form-data="{"contentType":null}" capture

which allow the images browsing in mobile ;)

I didn't find the way ....

Thanks for your help !!

Stéphane

**Comment by nonissue** [comment]
It's not very clear what you're asking for. If you could provide some code and a more specific explanation someone might be able to help.

**Comment by pilere** [comment]
of course, most of the message was hidden ... I took off the braces, you should understand what I meant ...

Stéphane

**Comment by 9935c101ab17a66** [comment]
I have no idea what you're trying to do. You won't get any help with just abstract snippets of what you are working on, and without being clear about your goals. 

Is this a bug with meteor-uploads or do you just not understand how to implement the feature you require? If you link to a repository someone might be able to help, but based on what you've provided nothing can be done.

**Comment by pilere** [comment]
I am sorry, my english is probably not good enough ...
let me try again, I use your perfect code in my meteor app, it's working fine BUT when I use my android phone to pick a file by clicking on the UPLOAD button, it trigger the wrong menu on the phone (with no icon for the media browser).
If I add manually another input field, with the word CAPTURE in it like this : 

&lt;input type="file" capture&gt;

it work perfectly ... so I would like to know how I can use your plugin to get the word capture added in the input field ... it may be already possible but I didn't manage to ....

is this clear enough ??

again I have to apologize, english is not my mother thong and it's easier for me to speak it than to write it !!

Stéphane

**Comment by tomitrescak** [comment]
Stephane, all clear now, unfortunately there is currently no easy way of doing that. I advise you to fork the upload-jquery package and add it to the template. If you create a PR and it will test well on the other platforms we can make it part of the core.

**Comment by pilere** [comment]
Hi tomi,

can you provide more specific direction on where to look for ? I just spend an hour trying to get the main DOM generation for the input line, with no luck ...
If I understand correctly, the templates folder is just for sample on how we can use the plugin ?

Thanks for your help !
Stéphane

**Comment by tomitrescak** [comment]
I would start here: https://github.com/tomitrescak/meteor-tomi-upload-jquery/blob/master/templates/upload.html#L16-L20

**Comment by pilere** [comment]
Le 21/10/2015 00:03, Tomas Trescak a écrit :
>
> I would start here: 
> https://github.com/tomitrescak/meteor-tomi-upload-jquery/blob/master/templates/upload.html#L16-L20
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/tomitrescak/meteor-uploads/issues/166#issuecomment-149715977>.
>
HI tomas,

I am such a dummy sometimes .... I did change the code and create a 
custom package, but then I found this page : 
http://stackoverflow.com/questions/21523544/html-file-input-control-with-capture-and-accept-attributes-works-wrong

and tried this :


{{> upload_bootstrap fileTypes="image/*"}}

which works PERFECTLY !
I deleted my github forks

THANKS AGAIN not only for your help but for this great package !

Stéphane

**Comment by tomitrescak** [comment]
Thanks a Lot for reporting this. I will make it part of the documentation. Have a lovely day!
